src/components/Bounty/BountyEntryUpsertForm.tsx

```diff
@@ -375,22 +360,25 @@ export function BountyEntryUpsertForm({ bountyEntry, bounty }: Props) {
                         What&apos;s this?
                       </Text>
                       <Text size="sm">
-                        Sample Files are files that you&apos;d like to share with Supporters
-                        reviewing your entry that might help them get a better idea of what
-                        you&apos;ve included with your bounty
+                        Bounty Files are the files that will be given to the Supporter when they
+                        award you the bounty
                       </Text>
                     </Stack>
                   </HoverCard.Dropdown>
                 </HoverCard>
               </Group>
             }
             orientation="vertical"
-            labelProps={{ size: 'xl' }}
+            labelProps={{ size: 'xl', sx: { display: 'inline-flex', gap: 8 } }}
             dropzoneProps={{
               maxFiles: 10,
-              accept: [...IMAGE_MIME_TYPE, ...ZIP_MIME_TYPE],
+              accept: acceptedFileTypes,
             }}
-            renderItem={(file, onRemove) => {
+            renderItem={(file, onRemove, onUpdate) => {
+              const metadata = (file.metadata ?? {}) as BountyEntryFileMeta;
+              // const currency = metadata.currency || getBountyCurrency(bounty);
+              // const unlockAmount = metadata.unlockAmount || 0;
```

Comment:
Same

Reply:
We are keeping this in case we are using it when we allow open entries